lib/Twig/Environment.php

```diff
@@ -789,6 +795,11 @@ public function hasExtension($name)
      */
     public function getExtension($name)
     {
+        if (isset($this->legacyExtensionNames[$name])) {
+            $name = $this->legacyExtensionNames[$name];
+            @trigger_error(sprintf('Referencing the "%s" extension by its name (defined by getName()) is deprecated since 1.26 and will be removed in Twig 2.0. Use the Fully Qualified Extension Class Name instead.', $name), E_USER_DEPRECATED);
+        }
+
```

Comment:
Should normalize name here.. `getExtension('\Something');` is equal to `getExtension('Something')`.

Perhaps rename `$name` to `$className` to clarify.